apps/preview/next/layouts/Showcases.tsx

```diff
@@ -86,10 +95,25 @@ export default function ShowcaseLayout({ children }: { children: ReactElement })
     set: (state) => setSearch((state as any).s),
   });
 
-  const isDocsRoute = !router.query.docs;
+  useEffect(() => {
+    if (!inIframe()) {
+      setIsDocsRoute(false);
+    } else {
+      window.parent.postMessage('loaded', '*');
+
+      window.addEventListener(
+        'message',
+        (e) => {
+          if (e.data === 'no-paddings') setNoPaddings(true);
+        },
+        false,
+      );
+    }
+  }, []);
+
   return (
     <div className="e-page-examples">
-      {isDocsRoute ? (
+      {!isDocsRoute ? (
```

Comment:
Did you mean to revers this condition?

Reply:
yes, probably should change variable name ;D